README.md

```diff
@@ -1,4 +1,80 @@
 # raincatcher-typescript
 Demo for new raincatcher architecture using typescript and DI
 
-See main JIRA issue at https://issues.jboss.org/browse/RAINCATCH-788
+This repository using [Lerna](https://lernajs.io/ to manage independent node.js modules. 
+
+## How to run
+
+```
+npm install -g lerna
+
+## Install dependencies
+npm install
+
+## Bootstrap application
+lerna boostrap
+
+npm start
+```
+
+
+## Basic workflow targetting the original separate repos
+
+1. Add the original remotes to your working copy
+```
+node scripts add-remotes
```

Comment:
these are only available here for the moment: https://github.com/feedhenry-raincatcher/raincatcher-mono/tree/master/scripts

Made to work with the original multi repos. Do you think we should go ahead and import all the supported modules + the custom scripts over to this repo?